### ISSUE TITLE
Training Code Request

### ISSUE THREAD

**Issue by qtlark** [compphoto/IntrinsicHDR#1]
Thank you for your great work!

I would be very grateful if you could share the training code.

**Comment by TQian1223** [comment]
Have you used HDR-VDP3 as an evaluation metric?

**Comment by qtlark** [comment]
> Have you used HDR-VDP3 as an evaluation metric?

https://sourceforge.net/projects/hdrvdp/files/hdrvdp/3.0.7/

**Comment by sebastian-dille** [comment]
Hi @TQian1223, 
yes, we're using the HDR-VDP3 metric to evaluate our work.
You can find the code using the link above (thanks @qtlark).
We are reporting the average JOD units corresponding to the "quality" task of the metric in our paper and the supplementary. 

Cheers

**Comment by sebastian-dille** [comment]
@qtlark we are currently not planning to release the full training code.
However, most details for the loss functions, etc are implemented in the reconstructor and refiner modules.

**Comment by TQian1223** [comment]
> Hi @TQian1223, yes, we're using the HDR-VDP3 metric to evaluate our work. You can find the code using the link above (thanks @qtlark). We are reporting the average JOD units corresponding to the "quality" task of the metric in our paper and the supplementary.
> 
> Cheers

Thank you very much for your help！ (＾▽＾)
